meilisearch-release-changelog/v1.14.0.md

```diff
@@ -0,0 +1,187 @@
+curl -H 'Content-Type: application/json' MEILISEARCH_URL/indexes/INDEX_UID/documents -d '{ "ids": ["cody", "finn", "brandy", "gambit"] }'
+```
+
+```jsonc
+{
+  "results": [
+    {
+      "id": "brandy",
+      "info": 13765493
+    },
+    {
+      "id": "finn",
+      "info": 35863
+    },
+    {
+      "id": "cody",
+      "info": 122263
+    },
+    {
+      "id": "gambit",
+      "info": 22222
+    }
+  ],
+  "offset": 0,
+  "limit": 20,
+  "total": 4
+}
+```
+
+> [!WARNING]  
+> Documents are *not* returned in the queried order. Non-existent documents are ignored.
+
+Done by @dureuill in [#5384](https://github.com/meilisearch/meilisearch/pull/5384).
+
+## Other improvements
+
+* Batch together `/documents` requests using either `PUT` or `POST` by @Kerollmops in [#5293](https://github.com/meilisearch/meilisearch/pull/5293)
+* Display timestamped internal indexing steps on the `/batches` route by @Kerollmops in [#5356](https://github.com/meilisearch/meilisearch/pull/5356) and [#5364](https://github.com/meilisearch/meilisearch/pull/5364)
+* Introduce `exhaustiveFacetCount` parameter to `/facet-search` route to retrieve an exhaustive facet count by @ManyTheFish in [#5369](https://github.com/meilisearch/meilisearch/pull/5369)
+* Reduce RAM consumption of arroy by @irevoire in https://github.com/meilisearch/arroy/pull/105
+* Experimental feature: Cache embeddings during search (Read more in the [feature discussion](https://github.com/orgs/meilisearch/discussions/818)) by @dureuill in [#5418](https://github.com/meilisearch/meilisearch/pull/5418)
+* Extend batch progress view to include indexing of vectors by @irevoire in [#5420](https://github.com/meilisearch/meilisearch/pull/5420)
+
+# Fixes 🐞
+
+* Geo update bug by @ManyTheFish in https://github.com/meilisearch/meilisearch/pull/5407
+* Isolate word fst usage by @ManyTheFish in https://github.com/meilisearch/meilisearch/pull/5415
+* Fix CI to work with merge queues by @Kerollmops in https://github.com/meilisearch/meilisearch/pull/5456
+* Support fetching the pooling method from the model configuration by @dureuill in [#5355](https://github.com/meilisearch/meilisearch/pull/5355)
+* Delete unused prefixes by @Kerollmops in [#5413](https://github.com/meilisearch/meilisearch/pull/5413)
+
+# Misc
+
+* Dependencies updates
+  * Bump Ubuntu in the CI from 20.04 to 22.04 by @Kerollmops in [#5338](https://github.com/meilisearch/meilisearch/pull/5338)
+  * Bump heed to v0.22 by @irevoire and @Kerollmops in [#5406](https://github.com/meilisearch/meilisearch/pull/5406)
+  * Bump ring to v0.17.14 to compile on old aarch64 by @Kerollmops in https://github.com/meilisearch/meilisearch/pull/5423
+  * Bump zip from 2.2.2 to 2.3.0 by @dependabot in https://github.com/meilisearch/meilisearch/pull/5426
+  * Update Charabia v0.9.3 by @ManyTheFish in https://github.com/meilisearch/meilisearch/pull/5454
+  * Update mini-dashboard to v0.2.19 version by @curquiza in https://github.com/meilisearch/meilisearch/pull/5436
```

Comment:
Same, here, can we describe the bug it fixes in `Bug fixes` section instead of making a line here?

Reply:
```suggestion
```